airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/SerializedBufferFactory.java

```diff
@@ -49,15 +49,18 @@
     LOGGER.info("S3 format config: {}", formatConfig.toString());
     switch (formatConfig.getFormat()) {
       case AVRO -> {
-        final Callable<BufferStorage> createStorageFunctionWithExtension = () -> createStorageFunctionWithoutExtension.apply(AvroSerializedBuffer.DEFAULT_SUFFIX);
+        final Callable<BufferStorage> createStorageFunctionWithExtension =
```

Comment:
do you have a different line width?

Reply:
I think this change was done by `./gradlew format`
(i wasn't editing this file in my ide)